### PR TITLE
add optional #:r/g/b to specify background color

### PR DESCRIPTION
**Pull request by david-vanderson**
Let me know if you'd rather specify r/g/b by flonum instead of by byte.  Thanks!

### CODE REVIEW

File: mode-lambda/backend/gl/util.rkt [hunk 1]
```diff
@@ -321,7 +321,7 @@
   (define (delayed-render static-arg ...)
     (define draw #f)
     (define last-glctx #f)
-    (λ (dyn-arg ...)
+    (λ (dyn-arg ... #:r (r 0) #:g (g 0) #:b (b 0))
```
Comment (jeapostrophe) [inline mode-lambda/backend/gl/util.rkt]: Can these be []s? Sorry for such an annoying change.

**Comment by jeapostrophe** [comment]
It's good, one tiny annoying change!

**Comment by david-vanderson** [comment]
No problem, done.

**Comment by jeapostrophe** [comment]
Thanks!

**Pull request merged**